exercises/binary-search/test/Tests.hs

```diff
@@ -0,0 +1,74 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+import Data.Array (Array, listArray)
+
+import BinarySearch (binarySearch)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "foundElement" $ for_ cases test
+  where
+    test Case{..} = it description assertion
+      where
+        assertion = binarySearch input value `shouldBe` expected
+
+
+data Case a = Case { description :: String
+                 , input       :: Array Int a
+                 , value       :: a
+                 , expected    :: Maybe Int
+                 }
```

Comment:
> I would indent this line two spaces further so that the comma lines up with the open bracket on the previous row.

same except now I'm talking about the closing bracket